.gitignore

```diff
@@ -1,3 +1,3 @@
 target
-tests/Cargo.lock
+**/Cargo.lock
```

Comment:
One annoying thing about `cross` is it leaves behind a `Cargo.lock` file in the crate's directory (and for that matter, doesn't use the existing crate resolution) but this change at least prevents them from being checked in.